curator/utils.py

```diff
@@ -1159,7 +1156,27 @@ def snapshot_running(client):
     :rtype: bool
     """
     try:
-        status = client.snapshot.status()['snapshots']
+        if aws_flag:
```

Comment:
How does this function get the `aws_flag` variable?  It won't keep state inside the `utils.py` file, as that's just a reference library.  That's why the tests are not passing:
```
ERROR            curator.cli                    cli:193  Failed to complete action: snapshot.  <class 'curator.exceptions.FailedExecution'>: Exception encountered.  Rerun with loglevel DEBUG and/or check Elasticsearch logs for more information. Exception: global name 'aws_flag' is not defined
```